crates/zk-por-core/src/merkle_proof.rs

```diff
@@ -19,17 +40,17 @@ impl MerkleProofIndex {
 
 /// Get the siblings index for the merkle proof of inclusion given a leaf index of a binary merkle sum tree.
 /// We get the parent index of a leaf using the formula: parent = index / 2 + num_leaves
-pub fn get_mst_siblings_index(global_index: usize, global_mst: &GlobalMst) -> Vec<usize> {
+pub fn get_mst_siblings_index(global_index: usize, cfg: &GlobalConfig) -> Vec<usize> {
```

Comment:
I think so too, but I dont want to load the global MST, it is heavy, we can probably refactor the global MST struct instead.

Reply:
> I think so too, but I dont want to load the global MST, it is heavy because it contains an entire vector of all the hashes, we can probably refactor the global MST struct instead.

agree